File: bachelor/sem3/rc/probleme1A/prob7/client7.php
<?php
$s = socket_create(AF_INET, SOCK_STREAM, 0);
socket_connect($s, "127.0.0.1", 3333);

$msg = readline("String for server: ");
socket_send($s, $msg, 100, 0);

$start_substring = readline("Start of wanted substring: ");
socket_send($s, $start_substring, 100, 0);

$substring_length = readline("Wanted substring length: ");
socket_send($s, $substring_length, 100, 0);

socket_recv($s, $substring_wanted, 100, 0);
echo "Received from server:: $substring_wanted\n"; 
?>

File: bachelor/sem3/rc/probleme1A/prob7/client7.py
import socket
import time

TCP_IP = "127.0.0.1"
TCP_PORT = 3333

c = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
c.connect((TCP_IP, TCP_PORT))

msg = str(input("String for server: "))
c.send(msg)

start_substring = str(input("Start of wanted substring: "))
c.send(start_substring)

#time.sleep(0.01)

c.send("3")
#substring_length = str(input("Wanted substring length: "))
#c.send(substring_length)

substring_wanted = c.recv(1024)
print("Received from server: ", substring_wanted)

c.close()
